django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -117,6 +126,10 @@
             header.find('.advanced-mode').click(function(){
                 // update autogenrated advanced json editor with new data
                 advanced_editor.set(JSON.parse(field.val()));
+
+                //set the advanced editor container to full screen mode
+                toggleFullScreen();
+
```

Comment:
same here, these blank lines are not needed, the code is readable enough to be understood without them